crates/turbo-trace/src/main.rs

```diff
@@ -30,7 +30,7 @@ fn main() -> Result<(), PathError> {
         .map(|f| AbsoluteSystemPathBuf::from_unknown(&abs_cwd, f))
         .collect();
 
-    let tracer = Tracer::new(abs_cwd, files, args.ts_config)?;
+    let tracer = Tracer::new(abs_cwd, files, args.ts_config);
```

Comment:
I'm not totally clear on why this change had anything to do with anything in this PR, but it wouldn't compile otherwise.

Reply:
alrighty. heh.